src/libstd/sync/rwlock.rs

```diff
@@ -330,6 +330,19 @@ impl<T: ?Sized + fmt::Debug> fmt::Debug for RwLock<T> {
     }
 }
 
+#[stable(feature = "std_guard_impls", since = "1.20")]
+impl<T: ?Sized + fmt::Display> fmt::Display for RwLock<T> {
+    fn fmt(&self, f: &mut fmt::Formatter) -> fmt::Result {
+        match self.try_read() {
+            Ok(guard) => write!(f, "RwLock {{ data: {} }}", guard),
```

Comment:
Is this supposed to be for `Display` or `Debug? This formatting suggests it's for `Debug` but this is an `impl` for `Display`